### DISCUSSION TITLE
TVA SPORT 2

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#750] [Q&A]
### Channel Name

TVA SPORT 2

### Channel ID

TVASports2.ca

### Country

CANADA

### Language

FRENCH

### Website

https://www.qub.ca/tvaplus/tva-sports-2

**Comment by Sphinxroot** [comment]
cable channel

**Comment by jphilippe2017** [comment]
Perhaps, but does the link still exist, anyone? 

Thank you!

**Comment by Sphinxroot** [comment]
nop exepted X-tream code but we cant put it in here

**Comment by jphilippe2017** [comment]
Ok, thank you for the answer on that! Appreciate your help!!

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19771
